packages/components/tests/dummy/app/templates/components/segmented-group.hbs

```diff
@@ -111,7 +111,16 @@
         <S.Button @color="secondary" @text="Button" />
         <S.Dropdown as |DD|>
           <DD.ToggleButton @color="secondary" @text="Dropdown" />
-          <DD.Interactive @href="#" @text="Dropdown Item" />
+          <DD.Checkbox name="checkbox-item-dropdown" @count="11">virtualbox</DD.Checkbox>
+          <DD.Checkbox name="checkbox-item-dropdown" @count="1" checked>vmware</DD.Checkbox>
```

Comment:
Added Inputs in the Dropdown list too. Looked into having one open by default (for VRT) as discussed, but unfortunately only the DisclosurePrimitive receives an `isOpen` argument, the MenuPrimitive (on which the Dropdown is built) [doesn't expose at the moment](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/menu-primitive/index.js#L12). If we ever enable that, it would be fairly easy to have VRT on such examples (and on the Dropdown component as well).